File: chatwoot/ruby/src/ListAllInboxesExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    response = ChatwootClient::InboxesApi.new.list_all_inboxes(
        0, # account_id
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling InboxesApi#list_all_inboxes: #{e}"
end

File: chatwoot/typescript-axios/src/ListAllInboxesExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
  // apiKey: "AGENT_BOT_API_KEY",
  // apiKey: "PLATFORM_APP_API_KEY",
});

new api.InboxesApi(configuration).listAllInboxes(
  0, // accountId
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling InboxesApi#listAllInboxes:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
